content/en/docs/concepts/services-networking/_index.md

```diff
@@ -7,44 +7,214 @@ description: >
+
+{{< figure src="/docs/images/k8net-localhost.drawio.svg" alt="k8s localhost" class="diagram-large" caption="Figure 2. Example of container communications  within the same pod using localhost:port number" >}}
+
+Your containers within a pod must coordinate port usage, but this is no different from processes in a VM.  
+
+How you implement this is a detail of the particular container runtime you employ in your environment.
+
+It is possible to request ports or call host ports on the node itself which forward packets to your pod. This is a very niche operation. How you implement this is a detail of the container runtime. The pod itself is blind to the existence or non-existence of host ports.
+
+### Pods on the same node
+
+You cluster deployment might include two or more pods running on the same node. You might also require communications between your pods on the same node. 
+
+Pods operate in their own namespace and have their own IP and MAC addresses. You can think of this as multiple distinct IP computers that want to talk with each other. Inter-pod networking on the same node is no different.
+
+This example of inter-pod networking on the same host requires:
+
+* Communication between the pod network namespaces and root network namespace. The establishment of a veth p2p links accomplishes this.
+
+* L2 or L3 networking between the pods. This example uses a virtual L2 bridge. Your implementation might use a different approach implemented in a network plugin.
+
+Figure 3 illustrates an example of two pods talking each other on the same node. 
+
+{{< figure src="/docs/images/k8net-PodSameHost03.drawio.svg" alt="k8s pods samehost" class="diagram-large" caption="Figure 3. Example of Pod 1 and Pod 2 on Node 1 communicating with each other" >}}
+
+
+ 
+### Pods on different nodes
+
+Your Kubernetes deployment might place pods on a network of nodes. With this pod topology, you can distribute your workloads across many pods to increase scale and resiliency. Inter-pod networking between distinct nodes becomes a crucial function of your cluster network. 
+
+
+#### Container network interface
+
+Network plugins support container networking across pods and nodes in your cluster environment. You can think of network plugins as the software responsible for container runtime interface attachment and detachment, pod IP address allocation and network function configuration such as tunnel setup, BGP and IP routing.  
+
+{{< note >}}
+Network plugins are generally referred to as CNI plugins.
+{{< /note >}}
+
+CNI plugins adhere to the specifications defined in the [container network interface](https://www.cni.dev/). The rational behind the development of the CNI was to avoid code duplication, simplify container network configuration and operations, and guarantee cluster compatibility. Most importantly, it offers the Kubernetes community a uni specification for building and deploying container networking functions in cluster networks. 
```

Comment:
```suggestion
CNI plugins adhere to the specifications defined in the [container network interface](https://www.cni.dev/).
The rationale behind the development of the CNI was to avoid code duplication, simplify container network
configuration and operations, and guarantee cluster compatibility.
Most importantly, CNI offers the Kubernetes community a unified specification for implementing
container networking functions in cluster networks.
```